README.md

```diff
@@ -205,7 +192,7 @@ docker build -f Dockerfile.allSdks . -t $MF_DOCKER_USER/mock-firebolt
 # Confirm your image was created
 docker images | grep mock-firebolt
 
-# Run the image, enabling just the core SDK (typical)
+# Run the image, enabling the core SDK (typical)
```

Comment:
Again, technically, we are enabling the manage SDK since the unified OpenRPC has the methods/messages it supports.

Reply:
Addressed